biojava-core/src/main/java/org/biojava/nbio/core/sequence/features/AbstractFeature.java

```diff
@@ -285,8 +285,17 @@ public void setQualifiers(HashMap<String, Qualifier> qualifiers) {
 
 	@Override
 	public void addQualifier(String key, Qualifier qualifier) {
-		// TODO Auto-generated method stub
-		Qualifiers.put(key, qualifier);
+		// Check for key. Update list of values
```

Comment:
e.g. this kind of stuff isn't necessary with implementations of ListMultimap, you just put the key, value and it uses a List for the value